tests/slack_sdk/web/mock_web_api_server.py

```diff
@@ -115,7 +115,7 @@ def _handle(self):
                     return
                 if "ratelimited" in pattern:
                     self.send_response(429)
-                    self.send_header("Retry-After", 1)
+                    self.send_header("retry-after", 1)
```

Comment:
Changed the mock API server's behavior -- with this, the existing test assets detect the lack of the `Retry-After` header